tests/e2e/performance/test_fio_benchmark.py

```diff
@@ -345,9 +345,12 @@ def cleanup(self):
         for line in pvs_list:
             pv, ns = line.split(" ")
             pv = pv.replace("'", "")
-            if ns == constants.RIPSAW_NAMESPACE:
-                log.info(f"Going to delete {pv}")
-                run_command(f"oc delete pv {pv}")
+            try:
```

Comment:
iiuc, pv deletion doens't need to be bound by namespace. 
in any case, wouldn't this could simply be 
`oc delete pv {pv} -n constants.RIPSAW_NAMESPACE`